core/src/main/java/org/elasticsearch/search/aggregations/pipeline/bucketmetrics/stats/extended/ExtendedStatsBucketParser.java

```diff
@@ -61,4 +52,14 @@ protected ExtendedStatsBucketPipelineAggregatorBuilder buildFactory(String pipel
     public ExtendedStatsBucketPipelineAggregatorBuilder getFactoryPrototype() {
         return ExtendedStatsBucketPipelineAggregatorBuilder.PROTOTYPE;
     }
+
+    @Override
+    protected boolean token(XContentParser parser, QueryParseContext context, String field,
+                            XContentParser.Token token, Map<String, Object> params) throws IOException {
+        if (context.parseFieldMatcher().match(field, SIGMA) && token.isValue()) {
```

Comment:
Yes but I think we should restrict this parameter to only numbers since we can't parse any string into a value of sigma, we should also be consistent with how sigma is parsed in the `extended_stats` aggregation which checks for `XContentParser.Token.VALUE_NUMBER`.